Packs/Whois/Integrations/Whois/Whois.py

```diff
@@ -8500,19 +8500,28 @@ def ip_command(ips, reliability):
 
 
 def whois_command(reliability):
-    query = demisto.args().get('query')
-    is_recursive = argToBoolean(demisto.args().get('recursive'))
+    args = demisto.args()
+    query = args.get('query')
+    is_recursive = argToBoolean(args.get('recursive'))
+    verbose = argToBoolean(args.get('verbose'))
     for query in argToList(query):
         domain = get_domain_from_query(query)
         whois_result = get_whois(domain, is_recursive=is_recursive)
         md, standard_ec, dbot_score = create_outputs(whois_result, domain, reliability, query)
         dbot_score.update({Common.Domain.CONTEXT_PATH: standard_ec})
+
+        if verbose: 
+            demisto.info('Verbose response')
```

Comment:
Might be worth adding the query in the log statement so it can be traced more easily.

Reply:
Done